charts/common/templates/workflowtemplate.yaml

```diff
@@ -33,7 +33,7 @@ spec:
     container:
       image: alpine:latest
       command: [sh, -c]
-      args: ["echo Image:{{`{{workflow.parameters.image}}`}} Tag:{{`{{workflow.parameters.tag}}`}} Command:{{`{{workflow.parameters.command}}`}}"]
```

Comment:
Just out of curiosity, have you tried making `echo` a command instead of passing it to `sh -c` ?

Reply:
I have not, I can try it out